src/swap/foxExchange.ts

```diff
@@ -1,29 +1,11 @@
-import {
-  asBoolean,
-  asEither,
-  asNull,
-  asNumber,
-  asObject,
-  asOptional,
-  asString,
-  asUnknown
-} from 'cleaners'
 import {
```

Comment:
Do we need to disable these in the gui too?

Reply:
eventually yes. But for now they will just never return a quote. Safe deprecation since our partners use these plugins in their app too.